aspnetcore/mvc/models/file-uploads/samples/2.x/SampleApp/Pages/Index.cshtml

```diff
@@ -0,0 +1,109 @@
+﻿@page
+@model IndexModel
+@using System.Net
+@{
+    ViewData["Title"] = "Upload Files Sample";
+}
+
+<h1>Files in the database</h1>
+
+@if (Model.DatabaseFiles.Count == 0)
+{
+    <p>
+        No files are available. Visit one of the file upload scenario pages to upload one or more files.
+    </p>
+}
+else
+{
+    <table>
+        <thead>
+            <tr>
+                <th></th>
+                <th>
+                    @Html.DisplayNameFor(model => model.DatabaseFiles[0].UntrustedName) /
+                    @Html.DisplayNameFor(model => model.DatabaseFiles[0].Note)
+                </th>
+                <th>
+                    @Html.DisplayNameFor(model => model.DatabaseFiles[0].UploadDT)
+                </th>
+                <th>
+                    @Html.DisplayNameFor(model => model.DatabaseFiles[0].Size)
+                </th>
+                <th>
+                    <code>FileStreamResult</code> from database
+                </th>
+            </tr>
+        </thead>
+        <tbody>
+            @foreach (var file in Model.DatabaseFiles) {
+                // Don't display the untrusted file name in the UI. HTML-encode the value.
+                var fileName = WebUtility.HtmlEncode(file.UntrustedName);
+                <tr>
+                    <td>
+                        <a asp-page="./DeleteDbFile" asp-route-id="@file.Id">Delete</a>
+                    </td>
+                    <td>
+                        <b>@Html.DisplayFor(modelItem => fileName)</b><br>
```

Comment:
`@fileName`

Reply:
Can I do this? ... and drop the encoding on Line 40? ...

```cshtml
<td>
    <b>@file.UntrustedName</b><br>
    @Html.DisplayFor(modelItem => file.Note)
</td>
```